mtp_bank_admin/settings/base.py

```diff
@@ -284,6 +284,15 @@
 CLOUD_PLATFORM_MIGRATION_MODE = os.environ.get('CLOUD_PLATFORM_MIGRATION_MODE', '')
 CLOUD_PLATFORM_MIGRATION_URL = os.environ.get('CLOUD_PLATFORM_MIGRATION_URL', '')
 
+# general ledger account code for prisoner monies holding bank account
+PRISONER_MONEY_HOLDING_ACCOUNT = '1841102059'
+
+# global setting to turn on all November 2 HMPPS policy changes
+NOVEMBER_SECOND_CHANGES_LIVE = os.environ.get('NOVEMBER_SECOND_CHANGES_LIVE', '').lower() in ('1', 'true')
+if NOVEMBER_SECOND_CHANGES_LIVE:
+    # new bank account will be in use from this date
+    PRISONER_MONEY_HOLDING_ACCOUNT = '1841102092'
```

Comment:
Do you think that we could mistakenly look at this and miss the second assignment to `PRISONER_MONEY_HOLDING_ACCOUNT` which could confuse any debugging if it came to be needed.

Given this should be very short lived, I'm sure we won't but for reasons like this I think we need to remember to pull all the feature flag settings out as soon as possible after 2nd November.

Reply:
yeah, it'll be gone soon and it's not even reading the code from the environment. i considered putting the code into mtp-deploy, to be consistent with other settings, but this seemed easier given that the old code had already been hard-coded for years